_templates/event.php

```diff
@@ -9,7 +9,7 @@
 
 $l10n->set_domain('layout');
 
-const EVENT_NAME = 'odin 6.0 release';
+const EVENT_NAME = 'juno 5.0 release';
```

Comment:
This seems like an oops

Reply:
Nope, I was explicitly setting it back to the value before this PR. This const controls whether or not the flip clock appears, and doesn't have anything to do with the toast. :wink: